crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/config/ResourceServerConfiguration.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.justice.laa.crime.contribution.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationEntryPoint;
+import org.springframework.security.oauth2.server.resource.web.access.BearerTokenAccessDeniedHandler;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.access.AccessDeniedHandler;
+
+@Configuration
+@Order(1)
+@EnableWebSecurity
+public class ResourceServerConfiguration {
+
+    public static final String API_PATH = "/api/**";
```

Comment:
Since these are only used inside this class we can make these constants private. (helps to limit variable scope to only what is required as also promotes better encapsulation)